services/tier-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/TierService.java

```diff
@@ -29,60 +29,36 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 public class TierService implements RequestHandler<Map<String, Object>, APIGatewayProxyResponseEvent> {
     private static final Logger LOGGER = LoggerFactory.getLogger(TierService.class);
     private static final String TABLE_NAME = System.getenv("TABLE_NAME");
     private static final Map<String, String> CORS = Map.of("Access-Control-Allow-Origin", "*");
-    // private static final String DATASTORE_IMPL = System.getenv("DATASTORE_IMPL");
-
-    // private static final String API_GATEWAY_HOST = System.getenv("API_GATEWAY_HOST");
-    // private static final String API_GATEWAY_STAGE = System.getenv("API_GATEWAY_STAGE");
-    // private static final String API_TRUST_ROLE = System.getenv("API_TRUST_ROLE");
-    // private static final String SYSTEM_API_CALL_DETAIL_TYPE = "System API Call";
-    // private static final String SYSTEM_API_CALL_SOURCE = "saas-boost";
 
     private final TierDataStore store;
 
-    // TODO EventBridge will eventually be needed to send tier definition change events
-    // private final EventBridgeClient eventBridge;
-
-    /*
-     * getTiers, getTier, createTier, updateTier, deleteTier
-     * each requires access to the backing DDB datastore
-     *
-     * DDB table needs to be passed as environment variable
-     *
-     * getTiers, getTier, createTier, updateTier, deleteTier
-     * are all kind of ONLY DDB datastore actions
-     * except maybe update.
-     * either way, we should have a Tier object. JSON serializable. and DDB serializable.
-     */
     public TierService() {
         final long startTimeMillis = System.currentTimeMillis();
         if (Utils.isEmpty(TABLE_NAME)) {
             throw new IllegalStateException("Missing environment variable TIERS_TABLE");
```

Comment:
TIERS_TABLE or TABLE_NAME? Prefer `TIERS_TABLE` to match the way we do it in the other services